dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -29,6 +29,9 @@
   %script{src: "#{static_asset_base_path}js/ace/ext-language_tools.js"}
   %script{src: asset_path('js/droplet/droplet-full.js')}
   %script{src: asset_path('js/tooltipster/jquery.tooltipster.js')}
+  %script{src: asset_path('js/fileupload/jquery.iframe-transport.js')}
+  %script{src: asset_path('js/fileupload/jquery.fileupload.js')}
+
```

Comment:
We end up using the plugin across all browsers. The plugin then uses AJAX or iframe submission (I believe) depending on support. It's possible we could include iframe-transport only on IE9.